pkg/lightning/restore/check_info.go

```diff
@@ -14,20 +14,31 @@
 package restore
 
 import (
+	"bytes"
 	"context"
 	"fmt"
+	"io"
 	"path/filepath"
 	"reflect"
 	"strings"
 
+	"github.com/pingcap/tidb/table/tables"
+
+	"github.com/pingcap/br/pkg/lightning/backend/kv"
+	"github.com/pingcap/br/pkg/lightning/log"
+	verify "github.com/pingcap/br/pkg/lightning/verification"
+
+	"github.com/pingcap/failpoint"
+
+	"github.com/pingcap/br/pkg/lightning/mydump"
```

Comment:
why can't you sort the import into 3 groups like all existing files?

Reply:
I wrote a script that sorts all imports into 3 groups. 
```bash
#!/bin/bash

# Remove blank lines of imports.
find . -type f -name "*.go"  | grep -v pb.go | xargs  sed -i ":a;N;\$!ba;s/\"\n\n\t\"/\"\n\t\"/g"
# Sort all imports into 3 groups.
make static # ignore the output errors
```
@kennytm Do you think it is necessary to check it in CI?

Reply:
@gozssky yeah we could put it in `make static` (together with `gofumports`)

btw what happened to our Go style guide 🤣 https://github.com/pingcap/style-guide/tree/master/docs/go